helm-fullstack-multiregion.yml

```diff
@@ -0,0 +1,507 @@
+---
+include:
+  - remote: "https://raw.githubusercontent.com/jobtome-labs/ci-templates/v3.10.0/helm-fullstack-quality.yml"
+
+# EUROPE
+
+deploy:helm:api:production:europe:
+  extends: .deploy:production:helm
+
+  variables:
+    GOOGLE_KEY: ${GOOGLE_KEY_API_PRODUCTION_EUROPE}
+    CLUSTER_NAME: ${CLUSTER_NAME_PRODUCTION_EUROPE}
+    CLUSTER_ZONE: ${CLUSTER_ZONE_PRODUCTION_EUROPE}
+    DOMAIN: ${DOMAIN_API_PRODUCTION_EUROPE}
+    SECRET_YAML: ${SECRET_YAML_API_PRODUCTION_EUROPE}
+    NAMESPACE: ${NAMESPACE_API_PRODUCTION_EUROPE}
+    ENVIRONMENT: production
+    ENVIRONMENT_NAME: production-europe
+    ENVIRONMENT_PATH: ${ENVIRONMENT_PATH_API_EUROPE}
+
+  before_script:
+    - |
+
+      # CHECK VARIABLES PHASE
```

Comment:
is not possible to create something like this 

```bash
 TO_CHECK_VARIABLES=("ENVIRONMENT_NAME" "CLUSTER_NAME_QUALITY")
 for var in ${TO_CHECK_VARIABLES[@]}; do .....
 ```
 and export the `for` script instead of repeating each time the same script in all the `before_scripts` ?

Reply:
Yep, it's definitely possible.
Though, a major refactoring on CI Templates will happen in the new future. Will take your suggestion into consideration then since there are multiple templates with the same approach.
Thank you!

Reply:
Yep, it's definitely possible.
Though, a major refactoring on CI Templates will happen in the new future. Will take your suggestion into consideration then since there are multiple templates with the same approach.
Thank you!

Reply:
In regards to your below question, it's not complicated but it takes some meticulous effort and time for testing, as per the current CI Templates.
There are some ideas we are planning to take into account for the next major refactoring of the templates. One of them is to use dynamic templates generation for such situations.